core/forms.py

```diff
@@ -143,3 +143,15 @@ class Meta:
         fields = "__all__"
 
     updating_user_field_name = "uploader"
+
+class AccountForm(ModelForm):
+    class Meta:
+        model = User
+        fields = ['username', 'first_name', 'email']
+    
```

Comment:
It's possible to replace the 'first_name' field label by custom text in a [modelform's meta options](https://docs.djangoproject.com/en/3.2/topics/forms/modelforms/#overriding-the-default-fields). For instance, we can change it to match the field name to how it shows up when a user registers:
```suggestion
    labels = {
        'first_name': 'Real Name',
    }
        
```